wandb/sdk/launch/runner/local.py

```diff
@@ -135,51 +134,22 @@ def run(self, launch_project: LaunchProject) -> Optional[AbstractRun]:
         # In synchronous mode, run the entry point command in a blocking fashion, sending status
         # updates to the tracking server when finished. Note that the run state may not be
         # persisted to the tracking server if interrupted
-        if synchronous:
-            command_args += get_entry_point_command(
-                entry_point, launch_project.override_args
-            )
-            with open(
-                os.path.join(launch_project.aux_dir, DEFAULT_CONFIG_PATH), "w"
-            ) as fp:
-                json.dump(launch_project.launch_spec, fp)
-            command_str = command_separator.join(command_args)
-
-            wandb.termlog(
-                "Launching run in docker with command: {}".format(
-                    re.sub(r"WANDB_API_KEY=\w+", "WANDB_API_KEY", command_str)
-                )
-            )
-            run = _run_entry_point(command_str, launch_project.project_dir)
-            run.wait()
-            return run
-        # Otherwise, invoke `wandb launch` in a subprocess
-        raise LaunchError("asynchronous mode not yet available")
-
-
-def _run_launch_cmd(cmd: List[str]) -> "subprocess.Popen[str]":
-    """Invoke ``wandb launch`` in a subprocess, which in turn runs the entry point in a child process.
-
-    Arguments:
-    cmd: List of strings indicating the command to run
-
-    Returns:
-        A handle to the subprocess. Popen launched to invoke ``wandb launch``.
-    """
-    final_env = os.environ.copy()
-    # Launch `wandb launch` command as the leader of its own process group so that we can do a
-    # best-effort cleanup of all its descendant processes if needed
-    if sys.platform == "win32":
-        return subprocess.Popen(
-            cmd,
-            env=final_env,
-            universal_newlines=True,
-            creationflags=subprocess.CREATE_NEW_PROCESS_GROUP,
```

Comment:
Don't we still need this for windows?

Reply:
the entire _run_launch_cmd fn is unused, _run_entry_point (which is used) has its own windows handling

Reply:
Got it